user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserOperator.java

```diff
@@ -64,10 +67,13 @@
     public KafkaUserOperator(Vertx vertx,
```

Comment:
I know that some parameters were missing even before this PR but can we update the Javadoc now, wdyt?